src/pretix/base/models/event.py

```diff
@@ -528,12 +528,19 @@ def active_subevents(self):
 
     @property
     def active_future_subevents(self):
+        ordering = self.settings.get('frontpage_subevent_ordering', default='date_ascending', as_type=str)
+        orderfields = {
+            'date_ascending': ('date_from', 'name'),
+            'date_descending': ('-date_from', 'name'),
+            'name_ascending': ('name', 'date_from'),
+            'name_descending': ('-name', 'date_from'),
+        }[ordering]
```

Comment:
Using this inline dictionary now. There's still two places one would have to edit to add a new sorting method.